src/Halogen/Svg/Attributes.purs

```diff
@@ -236,6 +239,15 @@ points = attr (H.AttrName "points") <<< Array.intercalate " " <<< map (\(Tuple x
 
 pathLength :: forall r i. Number -> IProp (pathLength :: Number | r) i
 pathLength = attr (H.AttrName "pathLength") <<< show
+ 
+patternContentUnits :: forall r i. String -> IProp (patternContentUnits :: String | r) i
+patternContentUnits = attr (H.AttrName "patternContentUnits")
+
+patternTransform :: forall r i. String -> IProp (patternTransform :: String | r) i
+patternTransform = attr (H.AttrName "patternTransform")
```

Comment:
Rather than a String, should this take an Array of Strings and then the implementation can use `Array.intercalate " "` on that array?